guides/micronaut-logstash/micronaut-logstash.adoc

```diff
@@ -0,0 +1,58 @@
+include::{commondir}/common-header.adoc[]
+
+
+
+include::{commondir}/common-requirements.adoc[]
+
+include::{commondir}/common-completesolution.adoc[]
+
+
+== Install Logstash
+
+https://www.elastic.co/logstash/[Logstash]:
+
+____
+Logstash is a free and open server-side data processing pipeline that ingests data from a multitude of sources, transforms it, and then sends it to your favorite "stash."
+____
+
+* https://www.elastic.co/downloads/logstash[Download and unzip Logstash]
+* Prepare a https://www.elastic.co/guide/en/logstash/current/configuration.html[`logstash.conf` config file]
+* Run `bin/logstash -f logstash.conf`
```

Comment:
Even if we want to keep this I think we should provide the `docker run ...` command to execute Logstash in a Docker container.

Reply:
Note, this is a draft. It is not a finished guide. I am just dumping some experiments in a draft PR.

Reply:
:+1: